plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -746,6 +873,9 @@ bls12_381_finalVerify (BuiltinBLS12_381_MlResult a) (BuiltinBLS12_381_MlResult b
 CONVERSION
 -}
 
+-- | Converts the given integer to a bytestring. The first argument specifies
```

Comment:
This [fails if the target length is greate than 8192](https://github.com/IntersectMBO/plutus/blob/c86aca85c4104114b680108640ab6e2bf5c00f2c/plutus-core/plutus-core/src/PlutusCore/Bitwise.hs#L49).